### ISSUE TITLE
Rename Pop-up blocker gpo

### ISSUE THREAD

**Issue by htcfreek** [mozilla/policy-templates#896]
Languag: german

The naming of the popuo blocker gpo is confusing: It is called `pop-up-blocker deaktivieren` and has to set to `aktivieren` to block popups which means "enable the disabling of the feature" to block popups.

I suggest to remove the word `deaktivieren` from the gpo name

**Comment by mkaply** [comment]
The English for this is "Block pop-ups from websites:

So removing deaktivieren doesn't seem correct.

Involving one of our German translators for suggestions.
@vossni

**Comment by htcfreek** [comment]
> The English for this is "Block pop-ups from websites:
> 
> So removing deaktivieren doesn't seem correct.
> 
> Involving one of our German translators for suggestions.
> @vossni 

Then the correct translation will be `Pop-Ups von Webseiten blockieren`

**Comment by mkaply** [comment]
Do you mind submitting a pull request for this?

https://github.com/mozilla/policy-templates/blob/master/windows/de-DE/firefox.adml#L359

If not I'll take care of it.

**Comment by htcfreek** [comment]
> Do you mind submitting a pull request for this?
> 
> https://github.com/mozilla/policy-templates/blob/master/windows/de-DE/firefox.adml#L359
> 
> If not I'll take care of it.

What time frame do I have?

**Comment by mkaply** [comment]
Whenever you get a chance. My next release is Tuesday or Wednesday next week.

**Comment by htcfreek** [comment]
> Whenever you get a chance. My next release is Tuesday or Wednesday next week.

Will try to do this weekend

**Comment by vossni** [comment]
> > The English for this is "Block pop-ups from websites:
> > So removing deaktivieren doesn't seem correct.
> > Involving one of our German translators for suggestions.
> > @vossni
> 
> Then the correct translation will be `Pop-Ups von Webseiten blockieren`

True, the current translation is incorrect. However, I think the correct translation would be: "Pop-ups von Websites blockieren" (with a small u) ;)

**Comment by htcfreek** [comment]
> > > The English for this is "Block pop-ups from websites:
> > > So removing deaktivieren doesn't seem correct.
> > > Involving one of our German translators for suggestions.
> > > @vossni
> > 
> > Then the correct translation will be `Pop-Ups von Webseiten blockieren`
> 
> True, the current translation is incorrect. However, I think the correct translation would be: "Pop-ups von Websites blockieren" (with a small u) ;)

But `Webseiten` or `Websites`?

**Comment by vossni** [comment]
Ah, my mistake. I would prefer "Webseiten".

**Comment by htcfreek** [comment]
@mkaply 
you can close thisafter the new release today
